nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -59,6 +59,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [v2rayA](https://v2raya.org), a Linux web GUI client of Project V which supports V2Ray, Xray, SS, SSR, Trojan and Pingtunnel. Available as [services.v2raya](options.html#opt-services.v2raya.enable).
 
+- [ramfetch](https://codeberg.org/o69mar/ramfetch), a "fetch" tool which displays memory info using /proc/meminfo.
```

Comment:
This is a new **package**, not a new **NixOS module**.  No need to add a release note.

Reply:
Ah that was a misunderstanding on my part. Thank you for clearing that up.